src/App.js

```diff
@@ -89,19 +91,24 @@ class App extends Component {
         this.getData(lang);
     }
 
+    onNavClick = (index) => {
+        currentCatIndexGlobal = index;
+        let obj = dataExtractor(index);
+        this.getData(obj.langArray[0], obj);
```

Comment:
again, what is obj?

Reply:
Same as above.. The dataExtractor() function returns an object which i have assigned to the obj variable

Reply:
please rename it to something more descriptive like chartData